### PR TITLE
Support Juno Error Codes and returning falsy values from functions

### PR DESCRIPTION
**Pull request by thebongy**

### CODE REVIEW

File: src/juno-node.ts [hunk 1]
```diff
@@ -202,10 +222,9 @@ export default class JunoModule {
 				type: ResponseTypes.FunctionResponse,
 				data: res || {}
 			});
-			return true;
 		} else {
 			// Function wasn't found in the module.
-			return false;
+			throw new JunoError(ErrorTypes.UnkownFunction);
```
Comment (rakshith-ravi) [inline src/juno-node.ts:227]: `Unknown`. Typo

Comment (thebongy) [reply src/juno-node.ts:227]: 10/10 timing

Comment (thebongy) [reply src/juno-node.ts:227]: Will fix on develop

Comment (rakshith-ravi) [reply src/juno-node.ts:227]: Scam

**Pull request merged**